types/registration.yaml

```diff
@@ -0,0 +1,29 @@
+ValidatorRegistration:
+  type: object
+  description: "The [`ValidatorRegistration`]() object from the Builder API spec."
+  properties:
+    fee_recipient:
+      allOf:
+        - $ref: 'primitive.yaml#/ExecutionAddress'
+        - description: "Address to receive fees from the block."
+    gas_limit:
+      allOf:
+        - $ref: "primitive.yaml#/Uint64"
+        - description: "Gas limit the validator desires to target."
+    timestamp:
+      allOf:
+        - $ref: 'primitive.yaml#/Uint64'
+        - description: "Unix timestamp of registration."
+    pubkey:
+      allOf:
+        - $ref: 'primitive.yaml#/Pubkey'
+        - description: "BLS public key of validator."
+
+SignedValidatorRegistration:
+  type: object
+  description: "The [`SignedValidatorRegistration`]() object from the Builder API spec."
```

Comment:
I think we should define these as SSZ in the `consensus-specs` style in https://github.com/ethereum/builder-specs

can just have a simple `spec.md` for now

with stuff like:

```python
class SignedValidatorRegistration(Container):
    ....
```

and then we have concrete links to toss into these empty `()`

Reply:
this is also where we can specify the signing schema https://github.com/ethereum/consensus-specs/pull/2884 et al.

Reply:
added in `ethereum/builder-specs`